lib/cru_lib/global_registry_methods.rb

```diff
@@ -8,8 +8,8 @@ module GlobalRegistryMethods
 
 
     included do
-      after_commit :push_to_global_registry
-      after_destroy :delete_from_global_registry
+      after_commit :push_to_global_registry, on: [ :create, :update ]
+      after_commit :delete_from_global_registry, on: :destroy
```

Comment:
Model attributes are still available after_commit just like they are after_destroy, right? The callback needs global_registry_id.